### PR TITLE
[READY] Species rework

### PR DESCRIPTION
**Pull request by AffectedArc07**
## About The Pull Request
This PR replaces all the Citadel species customisation snowflake crap and makes it TG defaults. This allows us to add in new species a lot easier, cuts down on icon files, etc. Citadel IPCs have been removed as they are just human reskins, I plan to port Ziiro's oracle IPC implementation at some point.

## Why It's Good For The Game
Our current species system was flawed in many ways, lack of maintainability, some accessories should be pickable, some shouldnt, etc. This allows species in our code to work a lot better.

## Changelog
:cl: AffectedArc07
del: Removed Citadel IPCs, good ones are coming soon
tweak: Completely refactored species
/:cl:

### CODE REVIEW

File: code/modules/surgery/bodyparts/bodyparts.dm [hunk 1]
```diff
@@ -443,6 +403,8 @@
 			limb.icon_state = "[animal_origin]_[body_zone]"
 		return
 
+	var/icon_gender = (body_gender == FEMALE) ? "f" : "m" //gender of the icon, if applicable
```
Comment (CornMyCob) [inline code/modules/surgery/bodyparts/bodyparts.dm]: ONLY TWO????????????????

File: code/modules/mob/living/carbon/human/species_types/angel.dm [hunk 2]
```diff
@@ -142,4 +142,4 @@
 		H.movement_type &= ~FLYING
 		override_float = FALSE
 		H.pass_flags &= ~PASSTABLE
-		H.CloseWings()
\ No newline at end of file
+		H.CloseWings()
```
Comment (CornMyCob) [inline code/modules/mob/living/carbon/human/species_types/angel.dm]: huh

File: code/modules/surgery/bodyparts/bodyparts.dm (hunk 1 above)
Comment (AffectedArc07) [reply code/modules/surgery/bodyparts/bodyparts.dm]: This isnt tumblr

File: code/modules/mob/living/carbon/human/species_types/angel.dm (hunk 2 above)
Comment (AffectedArc07) [reply code/modules/mob/living/carbon/human/species_types/angel.dm]: The `EditorConfig` made me put a newline at the end

**Pull request merged**